AspNetCore.CacheOutput/CacheOutputAttribute.cs

```diff
@@ -82,6 +101,13 @@ public int SharedTimeSpan
         /// </summary>
         public bool Private { get; set; }
 
+        /// <summary>
+        /// Corresponds to CacheControl Public HTTP header. The "public" response directive indicates that any cache 
+        /// MAY store the response, even if the response would normally be non-cacheable or cacheable only within 
+        /// a private cache.
+        /// </summary>
+        public bool Public { get; set; }
```

Comment:
Hey, thanks for updating the PR - I'm sorry I didn't see a notification! The only issue I saw with this is that the old version used to hardcode "private" in the header: https://github.com/Iamcerba/AspNetCore.CacheOutput/pull/33/files#diff-951f5d08d226b9550dc3e112f60d66d73081165fcea438f52bb9ad2df44516f8L395

Although this feels like the correct contract - I worry about backwards compatibility. Anyone who hasn't been setting `Private = true` in their attribute settings won't have private set on the header anymore.

I guess another option would be to make https://github.com/Iamcerba/AspNetCore.CacheOutput/pull/33/files#diff-951f5d08d226b9550dc3e112f60d66d73081165fcea438f52bb9ad2df44516f8R423 read:

`Private = !Public`

Thoughts?

Reply:
I agree that better to default Private to true for backward compatibility and prevent content from being cached in a shared cache.
Regarding Private = !Public, this also works for me. We will keep flexibility (can't imagine for now when it could be required to use Private = True and Public = True at the same time, need to check what actually will be generated in a response in this scenario) and at the same time Private will be always defaulted to True unless Public is configured to be True explicitly.

Reply:
UPD:
Let's default Public to false in a property and keep everything else as is.
The property Private couldn't be defaulted to !Public in property definition. And defaulting it in the code like you suggested makes Private property useless at all. Removing Public at all like:
```
var cacheControl = new CacheControlHeaderValue
            {
                MaxAge = cacheTime.ClientTimeSpan,
                SharedMaxAge = cacheTime.SharedTimeSpan,
                MustRevalidate = MustRevalidate,
                Private = Private,
                Public = !Private,
                NoCache = NoCache
            };
```
is also doesn't look right because Public will always present in a response if Private is false.

If we provide in attribute:
```
[CacheOutput(Private=true, Public=true)]
```
then a response will be "Cache-Control: public, private".

Reply:
Ok, so to use public, you will need to provide:

`[CacheOutput(Private=false, Public=true)]`

Otherwise the public value won't have the expected effect?

Reply:
If we provide `[CacheOutput(Public=true)]`

Output will also be "Cache-Control: public, private".